homepage/templates/homepage/index.html

```diff
@@ -0,0 +1,146 @@
+            <div class="u-container-layout u-container-layout-4">
+              <h1 class="u-align-left u-custom-font u-font-roboto-slab u-text u-text-4">As a dog owner<br>
+              </h1>
+              <ul class="u-align-left u-text u-text-5">
+                <li>
+                  <span style="font-size: 1.875rem;">Create a custom profile for your puppy</span>
+                </li>
+                <li>
+                  <span style="font-size: 1.875rem;">Find the best daycare for your four-legged buddy, with
+                    filters!</span>
+                </li>
+                <li>
+                  <span style="font-size: 1.875rem;">Read / write reviews of daycares</span>
+                </li>
+                <li>
+                  <span style="font-size: 1.875rem;">Watch the history of visited daycares</span>
+                </li>
+                <li>
+                  <span style="font-size: 1.875rem;">Chat with daycare owners</span>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="u-clearfix u-section-2" id="sec-5cec">
+    <div class="u-clearfix u-expanded-width u-gutter-0 u-layout-wrap u-layout-wrap-1">
+      <div class="u-layout" style="">
+        <div class="u-layout-row" style="">
+          <div
+            class="u-align-left u-container-style u-layout-cell u-left-cell u-size-35 u-size-xs-60 u-white u-layout-cell-1"
+            src="">
+            <div class="u-container-layout u-container-layout-1">
+              <h1 class="u-text u-text-1">As a daycare owner<br>
+              </h1>
+              <ul class="u-text u-text-2">
+                <li>
+                  <span style="font-size: 1.875rem;">Manage your booking schedule</span>
+                </li>
+                <li>
+                  <span style="font-size: 1.875rem;">Set filters of approved dogs and services you provide</span>
+                </li>
+                <li>
+                  <span style="font-size: 1.875rem;">Read / write reviews of dog owners and their dogs&nbsp;</span>
+                </li>
+                <li>
+                  <span style="font-size: 30px;">Chat with dog owners</span>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div
+            class="u-align-center u-container-style u-image u-layout-cell u-right-cell u-size-25 u-size-xs-60 u-image-1"
+            src="" data-image-width="1200" data-image-height="996">
+            <div class="u-container-layout u-valign-middle u-container-layout-2" src=""></div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+
+  <footer class="u-align-center u-clearfix u-footer u-grey-80 u-footer" id="sec-e5b7">
+    <div class="u-clearfix u-sheet u-sheet-1">
+      <p class="u-small-text u-text u-text-variant u-text-1">Project Demo 1 - Team 1</p>
+    </div>
+  </footer>
+</body>
+
+</html>
```

Comment:
No newline at EOF error

Reply:
Apparently VSCode deletes empty lines at the end of HTML files.
Fixing it now.